src/main/java/innovimax/mixthem/operation/JoinOperation.java

```diff
@@ -0,0 +1,23 @@
+package innovimax.mixthem.operation;
+
+import innovimax.mixthem.arguments.ParamValue;
+
+public enum JoinOperation { 
+  DEFAULT_JOIN_COLUMN(1);
+
+  private ParamValue value;
```

Comment:
int

Reply:
Used as default value for parameters in Map.getOrDefault().
Another way is to create default value when used with ParamValue.createInt().